app/javascript/react/screens/App/Analytics/screens/AnalyticsDataCollection/AnalyticsDataCollection.js

```diff
@@ -1,16 +1,53 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Spinner, Button } from 'patternfly-react';
+import { Spinner, Button, Icon } from 'patternfly-react';
 
-const AnalyticsDataCollection = ({ onCancelClick }) => (
-  <React.Fragment>
-    <div className="large-spinner">
-      <Spinner loading size="lg" inline />
-      <h3>{__('Collecting inventory data')}</h3>
-    </div>
-    <Button onClick={onCancelClick}>{__('Cancel')}</Button>
-  </React.Fragment>
-);
+class AnalyticsDataCollection extends React.Component {
+  state = { mockLoaded: false };
+
+  componentDidMount() {
+    setTimeout(() => this.setState({ mockLoaded: true }), 3000);
```

Comment:
I'm having troubles understanding why we need the timeout here?

Reply:
This is just a temporary placeholder to show the spinner for 3 seconds, to demonstrate what it will look like when it's really running the data collector. I suppose it's a little unnecessary, I'll remove it if you want. But it'll be removed shortly anyway.

Reply:
OK, gotcha. Let's leave it as it is then.